python-kubectl/README.md

```diff
@@ -0,0 +1,114 @@
+# Voting App deployed with raw Kubernetes manifests
```

Comment:
Can we use another app instead of the voting app? I think we may have overused it in our samples and blogposts already.  How about the guest book from https://github.com/kubernetes/examples/tree/master/guestbook-go ?

Reply:
Or reuse https://kubernetes.io/docs/tutorials/stateless-application/guestbook/ ?